example/factorial.rb

```diff
@@ -8,7 +8,7 @@
 dataset  = 'factorial'
 
 def factorial(number)
-  return -1 * factorial(abs(number)) if number < 0
+  return -1 * factorial(abs(number)) if number.positive?
```

Comment:
negative?

Reply:
![stare](https://user-images.githubusercontent.com/517302/136434934-ca91d6f0-39be-4a31-8be2-ed856c0368f8.gif)

Uh … I even wrote negative? in the commit message for this change.

```suggestion
  return -1 * factorial(abs(number)) if number.negative?
```